fedbiomed/researcher/job.py

```diff
@@ -178,6 +180,13 @@ def __init__(self,
         self.validate_minimal_arguments(self._repository_args,
                                         ['training_plan_url', 'training_plan_class', 'params_url'])
 
+    def _load_training_plan_from_file(self):
+        sys.path.insert(0, self._keep_files_dir)
+        module = importlib.import_module(os.path.split(self._training_plan_file)[1][:-3])
```

Comment:
Rather than compute `os.path.split(self._training_plan_file)[1][:-3]` (makes assumption on file name structure, thus difficult to maintain) I would propose to pass a `training_plan_module` variable.

=> I push a patch for this